BigTable Fraud Detection test failed when upgrading libraries-bom

#7355 upgraded the core set of client libraries across many `java-docs-samples` directories, and the build failed on the test below.  This issue has been created to upgrade to the latest `com.google.cloud:libraries-bom` along with any other matching dependencies, and ensure that the test works.
```
Running StreamingPipelineTest
Tests run: 1, Failures: 1, Errors: 0, Skipped: 0, Time elapsed: 887.799 sec <<< FAILURE!
StreamingPipelineTest  Time elapsed: 887.798 sec  <<< FAILURE!
java.lang.AssertionError: expected:<1> but was:<0>
	at org.junit.Assert.fail(Assert.java:89)
	at org.junit.Assert.failNotEquals(Assert.java:835)
	at org.junit.Assert.assertEquals(Assert.java:647)
	at org.junit.Assert.assertEquals(Assert.java:633)

Results :

Failed tests:   StreamingPipelineTest: expected:<1> but was:<0>

Tests run: 1, Failures: 1, Errors: 0, Skipped: 0
```

--- a/bigtable/use-cases/fraudDetection/pom.xml
+++ b/bigtable/use-cases/fraudDetection/pom.xml
@@ -85,7 +85,7 @@
         <groupId>com.google.cloud</groupId>
         <scope>import</scope>
         <type>pom</type>
-        <version>25.4.0</version>
+        <version>26.1.4</version>
       </dependency>
     </dependencies>
   </dependencyManagement>

